File: Assignment5&7/program9.c
//write a program which accept number from user and return the count of digits in betwen 3 and 7..

#include<stdio.h>

int CountRange(int iNo)
{
    int iDigit = 0;
    int iCount = 0;
while(iNo>0)
{
    iDigit =((iNo % 3) && (iNo % 7));
    iNo = iNo / 10;
}
return iDigit;

}

int main()
{
    int iValue = 0;
    int iRet = 0;
    printf("Enter Number\n");
    scanf("%d",&iValue);

    iRet = CountRange(iValue);
    printf("%d\n",iRet);

    return 0;
}

File: Assinment30&31/program9.java
//write a program which accept one number from user and toggle 7th & 10th bit of that number. return modified number


import java.lang.*;
import java.util.*;
//  0000  0000   0000  0000  0000  0010  0100  0000
//  0       0      0    0      0    2      4   0

class Bitwise
{
    public int toggleBit(int iNo)
    {
        int iMask =0X00000240;      //576
        int iResult = 0;

        iResult = iNo ^ iMask;
        return iResult;

    }
}

class program9
{
    public static void main(String arg[])
    {
        Scanner sobj = new Scanner(System.in);
        System.out.println("Enter the Number");
        int iValue = sobj.nextInt();

        Bitwise bobj = new Bitwise();
        
        int iRet = bobj.toggleBit(iValue);

        System.out.println("Update Number is : "+iRet);


    }
}